spa/src/App.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <main id="app" class="h-screen flex flex-col">
+  <main id="app" class="h-screen flex flex-col" v-if="this.$route.meta.wrapper">
     <div
       class="flex flex-none flex-row bg-lines justify-between h-full"
       style="height: 70px;"
```

Comment:
Add the v-if into the v-if on this element as `v-if="$store.data.isUser && this.$route.meta.wrapper"` Also on line 27

Reply:
This is showing the wrong reference lines.

Lines to modify as discussed are line 6 and line 27 where there is currently `v-if="$store.data.isUser"`